great-idea/services.html

```diff
@@ -16,91 +16,117 @@
+eleifend. Mauris euismod facilisis iaculis.</p>
 
+<button>Learn More</button>
+</div>
 
+<div>
+<h3>UX / UI</h3>
 
-Web Development
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+<button>Learn More</button>
+</div>
 
-Learn More
+<div>
+<h3>Digital Marketing</h3>
 
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
+<button>Learn More</button>
+</div>
 
-iOS Development
+<div>
+<h3>Web Development</h3>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
-Learn More
+<button>Learn More</button>
+</div>
 
+<div>
+<h3>iOS Development</h3>
 
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
 
-Android Development
+<button>Learn More</button>
+</div>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+<div>
+<h3>Android Development</h3>
 
-Learn More
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
```

Comment:
So it looks like maybe it was because of copy pasting that the Great Idea webpage had those missing P tags? they all seem to be present here. Just be mindful of those kinds of small changes in the copied text. Make sure it is exactly what you want and are sure of it.